### PR TITLE
Replace breadcrumbs with Back links

### PR DESCRIPTION
**Pull request by fofr**
- Adds Back links to the easy use cases
- Removes add/edit logic, show the same page whether editing or not
- Fix default course choices

### CODE REVIEW

File: app/views/application/contact-details/address.njk [hunk 1]
```diff
@@ -10,9 +10,6 @@
 {% endif %}
 
 {% block pageNavigation %}
-  {{ govukBreadcrumbs({
-    items: applicationBreadcrumbItems
-  }) }}
 {% endblock %}
```
Comment (paulrobertlloyd) [inline app/views/application/contact-details/address.njk]: Empty `pageNavigation` block?

File: app/views/application/contact-details/select-address.njk [hunk 2]
```diff
@@ -1,5 +1,4 @@
 {% extends "_form.njk" %}
-
```
Comment (paulrobertlloyd) [inline app/views/application/contact-details/select-address.njk]: Stylistically, have been separating `extends` from the `set` statements that follow. Can we keep that?

File: app/views/application/contact-details/address.njk (hunk 1 above)
Comment (fofr) [reply app/views/application/contact-details/address.njk]: Something needs to go in here eventually. I just removed this so I could safely delete `applicationBreadcrumbItems`

Comment (paulrobertlloyd) [reply app/views/application/contact-details/address.njk]:
```suggestion
  {# TODO: Add back link #}
{% endblock %}
```

File: app/views/application/contact-details/select-address.njk (hunk 2 above)
Comment (fofr) [reply app/views/application/contact-details/select-address.njk]: Updated

File: app/views/application/contact-details/address.njk (hunk 1 above)
Comment (fofr) [reply app/views/application/contact-details/address.njk]: I've deleted it

**Pull request merged**